includes/Context.php

```diff
@@ -10,6 +10,9 @@
 
 namespace Google\Site_Kit;
 
+use AMP_Options_Manager;
+use AMP_Theme_Support;
```

Comment:
Just an idea (and very much my personal preference), but I typically don't use `use` statements for classes that _may_ not exist. I'm certainly open to keeping them, just wanted to have your thoughts.

Reply:
That's a good question. I generally `use` classes and add checks if necessary. In this case perhaps it seems more safe/clear to inline the class use and just prefix them with `\`, but I think this question likely would not come up if the class was in several levels of namespacing.

One might say as a principal that this kind of rule shouldn't be concerned with the number of levels of namespacing, but I'm not aware of any such rule or standard around this kind of thing.

In our case I don't have a strong opinion one way or the other, but if there were any more namespacing involved, I would opt for `use`.

Reply:
That's fair. We can leave it as is.